### ISSUE TITLE
Release Post Presentation

### ISSUE THREAD

**Issue by UO276188** [Arquisoft/lomap_en1b#120]
As we mentioned in the presentation, last night we tried to deploy a more "polished" version, that was not working because some test were failing. It was very sad we cannot show those improvements and as some teacher said it could be interesting.
So, as the teachers already saw the app working, we decide to make a new release v1.0 with all this issues fixed. It is all explained in the release itself [clicking here](https://github.com/Arquisoft/lomap_en1b/releases/tag/v1.0). 
To sum up: user experience has been improved with pop ups and automatically refetching, profile view is now working and those test failing are now fixed.

In case you want to check the app it is available on http://lomap.mariopdev.com

**Comment by UO276188** [comment]
We are sorry this didn't arrive in time for the presentation...

**Comment by UO276188** [comment]
Also there is a new option to deploy the application locally from the develop branch, as it is specified in the Readme file.
So the develop branch is now prepared to be deployed locally.

**Comment by pglez82** [comment]
Ok. We are going to ask you to have the application available for some time next week so we can check it again. 

I tag the other teachers so they know that you have uploaded this new funcionality.

@labra @cidinene @augustocristian
